101-azurefirewall-hub-spoke-demo/azuredeploy.json

```diff
@@ -4,16 +4,12 @@
     "parameters": {
         "adminUsername": {
             "type": "string",
-            "defaultValue": "azfw",
             "metadata": {
               "description": "Username for the Virtual Machine."
             }
           },
           "adminPassword": {
-            "type": "securestring",
-            "metadata": {
```

Comment:
revert this change (add the metadata property back)

Reply:
Added metadata description back in the template and removed from the parameters file.